elastic/config.go

```diff
@@ -126,9 +126,10 @@ type FieldNamesConfig struct {
 	EventIngested string `yaml:"event_ingested"`
 
 	// DNS, IP, HTTP, TLS
-	Timestamp string    `yaml:"timestamp"`
-	SrcIP     FieldPath `yaml:"src_ip"` // required
-	SrcPort   FieldPath `yaml:"src_port"`
+	Timestamp       string    `yaml:"timestamp"`
+	TimestampFormat string    `yaml:"timestamp_format"`
```

Comment:
I think it might be better to put this field not under FieldNames (because it's not a field name), but one level higher, under SearchConfig.